tests/onnxruntime/test_modeling_ort.py

```diff
@@ -706,3 +754,134 @@ def test_default_pipeline_and_model_device(self, *args, **kwargs):
         preprocessor = get_preprocessor(model_id)
         pp = pipeline("image-classification", model=onnx_model, feature_extractor=preprocessor)
         self.assertEqual(pp.device, onnx_model.device)
+
+
+class ORTModelForSeq2SeqLMIntegrationTest(unittest.TestCase):
+    SUPPORTED_ARCHITECTURES_WITH_MODEL_ID = {
+        "t5": "t5-small",
+        "bart": "facebook/bart-base",
+        "marian": "sshleifer/tiny-marian-en-de",
+    }
```

Comment:
i checked the support configs. can we add `BigBirdPegasus`, `LongT5`, `M2M100`, `MBART`

Reply:
Good point, I have added them